sonarts-core/tests/rules/noUnthrownErrorRule/noUnthrownErrorRule.lint.ts

```diff
@@ -0,0 +1,14 @@
+// OK
+foo(new Error());
+foo(TypeError);
+throw new Error();
+
+// NOK
+  new Error();
```

Comment:
@inverno error will be created even without `new`. I think we should cover this case as well

Reply:
@vilchik-elena I'll try and see what shows up with this one